src/main/java/com/nvidia/spark/rapids/jni/GpuTimeZoneDB.java

```diff
@@ -91,12 +94,21 @@ public static void cacheDatabaseAsync(int maxYear) {
     thread.start();
   }
 
-  public static synchronized void verifyDatabaseCached() {
+  private static synchronized void verifyDatabaseCachedSync() {
     if (timeZoneInfo == null) {
       throw new IllegalStateException("Timezone DB is not loaded, or the loading was failed.");
     }
   }
 
+  public static void verifyDatabaseCached() {
```

Comment:
I don't know how this API is used.
But since it is public, personally it would be better return a boolean instead or comment that it would throw out an exception when database is missing.

Reply:
`verifyDatabaseCachedSync` is a `static synchronized` method.  Normally it does not throw an exception. Normally it waits loading is done. And it's expected that exception occurs when database is missing.